rebar.lock

```diff
@@ -0,0 +1 @@
+[].
```

Comment:
It represents the actual exact dependencies used for the project. This is like a `yarn.lock` or `composer.lock`, for rebar3.

Here we don't have any dependencies so it is an empty list. So yes, in this sense having it or not does not change anything. But it may be considered standard to have it and it may cause issues to gitignore it if dependencies are added in the future.

In projects with no dependencies, [some](https://github.com/talentdeficit/jsx) put it, [some other](https://github.com/davisp/jiffy) not.

For my part, I will argue to put it anyway.